python/Create_Large_Test_Files.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Creating Custom Row HMDA Submission Test Files
+# 
+# This notebook demonstrates a class for creating custom row HMDA submission test files.
+
+# In[3]:
+
+
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+from custom_test_files import Custom_Test_Files
+lg = lar_gen() #Instantiates lar_gen class as lg. 
```

Comment:
What do you think about using a longer name in order to help future readers follow the code more easily?